solvers/ChambollePockPDSplit.py

```diff
@@ -44,7 +44,7 @@ def run(self, callback):
         sigma_w = 1.0 / (self.ratio * LA)
         # Init variables
         n, m = self.y.shape
-        u = np.zeros((n, m))
+        self.u = u = np.zeros((n, m))
```

Comment:
sure?
I'm not very familiar with chaining =, if it can be avoided it's better IMO

Reply:
`self.u` and `u` both need to be initialized for the solver to run correctly. [This syntax](https://github.com/wassimmazouz/benchmark_tv_1d/blob/6655e8bf3a48b7dab25b9e8eb77c2792e2d19a27/solvers/ADMM.py#L47) was used in `benchmark_tv_1d`.

I will make necessary changes then